dev/idm-am-common-user/idm/conf/router.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Given that there is no IG protecting IDM, you probably want to go back to the stock router.json rather than using this one (which has no router-authz.json entry)